src/lib.rs

```diff
@@ -309,11 +309,13 @@ impl Navigator {
     /// ```no_run
     /// use navigator_rs::{Navigator, PwmChannel};
     ///
-    /// let mut nav = Navigator::new();
-    /// nav.init();
-    /// nav.pwm_enable();
-    /// nav.set_pwm_freq_prescale(100); // sets the pwm frequency to 60 Hz
-    /// nav.set_pwm_channel_value(PwmChannel::Ch0, 2048); // sets the duty cycle to 50%
+    ///let mut nav = Navigator::new();
+    ///
+    ///nav.init();
+    ///nav.pwm_enable();
+    ///
+    ///nav.set_pwm_freq_prescale(99); // sets the pwm frequency to 60 Hz
+    ///nav.set_pwm_channel_value(PwmChannel::Ch0, 2048); // sets the duty cycle to 50%
```

Comment:
It should have a space before the first character of each line of the doc comment